web/views.py

```
@@ -2236,6 +2245,13 @@ def teacher_dashboard(request):
     # Get the teacher's storefront if it exists
     storefront = Storefront.objects.filter(teacher=request.user).first()
 
+    # Get user's membership subscription
+    subscription = None
+    try:
+        subscription = UserMembership.objects.get(user=request.user)
+    except UserMembership.DoesNotExist:
+        pass
```

Ruff (0.8.2)

Use `contextlib.suppress(UserMembership.DoesNotExist)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(UserMembership.DoesNotExist)`

(SIM105)